### ISSUE TITLE
Random NPE when closing inventory

### ISSUE THREAD

**Issue by Azodox** [DevNatan/inventory-framework#140]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### 🐛 Describe the bug

Hey, I'm having these random NPEs coming from nowhere, they sometimes occurs, sometimes not but for sure they occurs when I close an inventory (that I created) with E or Escape.
For the record, my view is a PaginatedView.
I don't know if it's a bug related to my code but here is the given error :

```
[11:52:25 ERROR]: Could not pass event InventoryCloseEvent to Core v0.2.0
java.lang.NullPointerException: Cannot invoke "me.saiintbrisson.minecraft.ViewContext.getRoot()" because "parent" is null
        at me.saiintbrisson.minecraft.CloseViewContext.<init>(CloseViewContext.java:20) ~[xmas-core-0.2.0.jar:?]
        at me.saiintbrisson.minecraft.ViewListener.onViewClose(ViewListener.java:141) ~[xmas-core-0.2.0.jar:?]
        at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[paper-api-1.19-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-api-1.19-R0.1-SNAPSHOT.jar:git-Paper-45]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-api-1.19-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:664) ~[paper-api-1.19-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_19_R1.event.CraftEventFactory.handleInventoryCloseEvent(CraftEventFactory.java:1533) ~[paper-1.19.jar:git-Paper-45]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleContainerClose(ServerGamePacketListenerImpl.java:2813) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleContainerClose(ServerGamePacketListenerImpl.java:2806) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundContainerClosePacket.handle(ServerboundContainerClosePacket.java:15) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundContainerClosePacket.a(ServerboundContainerClosePacket.java:10) ~[?:?]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$1(PacketUtils.java:51) ~[?:?]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.19.jar:git-Paper-45]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1358) ~[paper-1.19.jar:git-Paper-45]
        at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:183) ~[paper-1.19.jar:git-Paper-45]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1335) ~[paper-1.19.jar:git-Paper-45]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1328) ~[paper-1.19.jar:git-Paper-45]
        at net.minecraft.util.thread.BlockableEventLoop.runAllTasks(BlockableEventLoop.java:114) ~[?:?]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1462) ~[paper-1.19.jar:git-Paper-45]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1187) ~[paper-1.19.jar:git-Paper-45]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:302) ~[paper-1.19.jar:git-Paper-45]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

### ✔️ Expected behavior

Basically no error when I close my inventory.

### 👣 Steps to Reproduce

1. Create a custom view and open it to your player
2. Go in the game, open this view however you decided
3. Close this inventory with E (by default) or Escape
4. Try a couple times, as said before, sometimes the error shows, sometimes not.

### 💻 Platform

- [ ] Bukkit
- [ ] Sponge
- [X] Other

### ⭐ Version

1.19

### ✍️ Additional context

I'm running Paper.
